lib/decimal.ex

```diff
@@ -1391,6 +1391,26 @@ defmodule Decimal do
     tmp
   end
 
+  @doc """
+  Returns `true` when the given `decimal` has no significant digits after the decimal point.
+
+  ## Examples
+
+      iex> Decimal.integer?("1.00")
+      true
+
+      iex> Decimal.integer?("1.1000")
+      false
+  """
+  @spec integer?(t) :: boolean
+  def integer?(number) do
+    normalized = %Decimal{exp: exp} = number |> decimal() |> Decimal.normalize()
+    exp > 0 or number_after_dot(normalized) == 0
```

Comment:
I think below is equivalent but hopefully a little bit easier to read?

```suggestion
    number = decimal(number)
    normalized = normalize(number)
    number.exp > 0 or number_after_dot(normalized) == 0
```